xtp/src/libxtp/orbitals.cc

```diff
@@ -647,9 +650,17 @@ void Orbitals::ReadFromCpt(CheckpointReader r) {
 
   r(qm_energy_, "qm_energy");
   r(qm_package_, "qm_package");
+  try {
```

Comment:
read and write do not use the same variables, it only reads pm_localized_orbitals